src/validation.cpp

```diff
@@ -3040,7 +3047,7 @@ bool ContextualCheckBlock(const CBlock& block, CValidationState& state, const Co
     //   {0xaa, 0x21, 0xa9, 0xed}, and the following 32 bytes are SHA256^2(witness root, witness nonce). In case there are
     //   multiple, the last one is used.
     bool fHaveWitness = false;
-    if (VersionBitsState(pindexPrev, consensusParams, Consensus::DEPLOYMENT_SEGWIT, versionbitscache) == THRESHOLD_ACTIVE) {
+    if (VersionBitsState(pindexPrev, consensusParams, Consensus::DEPLOYMENT_SEGWIT2X, versionbitscache) == THRESHOLD_ACTIVE) {
```

Comment:
@jameshilliard Ok. I misread that, but still it would make it trivial for those that prefer SegWit only, to false flag SEGWIT2X. 

Maybe we should wait until the details of the HF activation are included, but if we accept and judge this repo as an implementation of the agreement (as I interpret it lock in of both HF and SegWit), separate signaling and service flag may be preferred.